apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/modules/components/Claimable.tsx

```diff
@@ -0,0 +1,646 @@
+                      <FormItem className="grow">
+                        <FormControl>
+                          <Input
+                            placeholder="0x..."
+                            {...field}
+                            disabled={!props.isOwnerAccount}
+                          />
+                        </FormControl>
+                        <FormMessage />
+                      </FormItem>
+                    )}
+                  />
+                  <ToolTipLabel label="Remove address">
+                    <Button
+                      variant="outline"
+                      className="!text-destructive-text bg-background"
+                      onClick={() => {
+                        allowListFields.remove(index);
+                      }}
+                      disabled={!props.isOwnerAccount}
+                    >
+                      <Trash2Icon className="size-4" />
+                    </Button>
+                  </ToolTipLabel>
+                </div>
+              ))}
+
+              {allowListFields.fields.length === 0 && (
+                <Alert variant="warning">
+                  <CircleAlertIcon className="size-5 max-sm:hidden" />
+                  <AlertTitle className="max-sm:!pl-0">
+                    No allowlist configured
+                  </AlertTitle>
+                </Alert>
+              )}
+            </div>
+
+            <div className="h-1" />
+
+            <div className="flex gap-3">
+              <Button
+                variant="outline"
+                size="sm"
+                onClick={() => {
+                  // add admin by default if adding the first input
+                  allowListFields.append({
+                    address: "",
+                  });
+                }}
+                className="gap-2"
+                disabled={!props.isOwnerAccount}
+              >
+                <PlusIcon className="size-3" />
+                Add Address
+              </Button>
+            </div>
+          </div>
+
+          <div className="flex justify-end">
+            <Button
+              size="sm"
+              className="min-w-24 gap-2"
+              disabled={updateMutation.isPending || !props.isOwnerAccount}
+              type="submit"
+            >
+              {updateMutation.isPending && <Spinner className="size-4" />}
+              Update
+            </Button>
+          </div>
+        </div>
+      </form>{" "}
+    </Form>
+  );
+}
+
+const mintFormSchema = z.object({
+  tokenId: z.number().optional(),
+  quantity: z.string().refine((v) => v.length > 0 && Number(v) >= 0, {
+    message: "Invalid quantity",
+  }),
+  recipient: addressSchema,
+});
+
+export type MintFormValues = z.infer<typeof mintFormSchema>;
+
+function MintNFTSection(props: {
+  mint: (values: MintFormValues) => Promise<void>;
+  isErc721: boolean;
+  isSequentialTokenIdInstalled: boolean;
+}) {
+  const form = useForm<MintFormValues>({
+    values: {
```

Comment:
the `mintFormSchema` is not hooked up to the form. Need to add 
`resolver: zodResolver(mintFormSchema)` here